script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -266,7 +266,7 @@ I was only here on business today. It may have dragged on for a while, but I got
 [sha:21a64ef5004a0837d353dc53d0139ed3889826a4]{
 -- Page 325, Offset 1987.
 -- ああ、この陰気な姿でせっかくの帰郷に水を差すのも忍びない。今日はこのあたりで失礼するよ」
-Aah, the last thing I want to do is spoil your homecoming with my gloomy appearance. Please, excuse me."
+Ahh, the last thing I want to do is spoil your homecoming with my gloomy appearance. Please, excuse me."
```

Comment:
I think all these should be just 'Ah'. Opinions please.

Reply:
I am wholly indifferent but it's somewhat nice to have it be a longer syllable in english too

Reply:
im fine either way i thought i had to replace the a with something :aichanteehee: